### PR TITLE
feat: logger for output & errors

### PR DESCRIPTION
**Pull request by ankithans**
## Description
- implement top level factory for commands
- logger for printing output and error
   - not a complete logger though, just implemented two methods
   - lot of useful methods & options can be added in logger
- use factory with logger in plugins
- import plugins into Host

want to validate, if this is a fine approach?

### CODE REVIEW

**Comment by ankithans** [comment]
@wtrocki

**Review by wtrocki — APPROVED** [review]

**Comment by wtrocki** [comment]
Good start

File: pkg/logging/logger.go [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package logging
+
+type Logger interface {
```
Comment (wtrocki) [inline pkg/logging/logger.go:3]: Docs please

File: pkg/logging/std_logger.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+		logger.outStream = os.Stdout
+	}
+	if logger.errStream == nil {
+		logger.errStream = os.Stderr
+	}
+
+	return
+}
+
+func (l *StdLogger) Output(args ...interface{}) {
```
Comment (wtrocki) [inline pkg/logging/std_logger.go:47]: Docs

**Pull request merged**